docs/scenarios_authoring/Spel.md

```diff
@@ -376,3 +376,11 @@ On the other hand, formatter created using `#DATE_FORMAT.formatter()` method wil
 - `#DATE_FORMAT.lenientFormatter('yyyy-MM-dd EEEE', 'PL')` - creates lenient version `DateTimeFormatter` using given pattern and locale
 
 For full list of available format options take a look at [DateTimeFormatter api docs](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/format/DateTimeFormatter.html).
+
+## Casting.
+
+When a type cannot be determined by parser, the type is presented as `Unknown`. When we know what the type will be on
+runtime, we can cast a given type, and then we can operate on the cast type.
+
+E.g. having a variable `obj` of a type: `List[Unknown]` and we know the elements are strings then we can cast elements
+to String: `#obj.![#this.canCastTo('java.lang.String') ? #this.castTo('java.lang.String') : null]`.
```

Comment:
I guess it will not be easy, but I will check it in next commit.

Reply:
Do you mean in the next PR?

Reply:
Also:
- We have `castToOrNull` now so it can be shortened
-  I think that it would be nice, to list each of these three methods in bullets with short explanation. Zibi would appreciate that

Reply:
Done